Consul.Test/AgentTest.cs

```diff
@@ -994,6 +994,17 @@ public async Task Agent_FilterChecks()
             Assert.Equal(checkName, actual.Values.First().Name);
         }
 
+        [Fact]
+        public async Task Agent_HostInfo()
+        {
+            var hostInfo = await _client.Agent.GetAgentHostInfo();
+            Assert.NotNull(hostInfo.Response.Host);
+            Assert.NotNull(hostInfo.Response.Memory);
+            Assert.NotNull(hostInfo.Response.CPU);
```

Comment:
Yeah, I tried it locally with version 1.9.17 and indeed, the CPU entry is missing in the repose. So lets remove the assertion for it.
```suggestion
```